kotlinx-coroutines-core/common/src/flow/operators/Delay.kt

```diff
@@ -33,7 +50,14 @@ import kotlin.time.*
  *     emit(5)
  * }.debounce(1000)
  * ```
- * produces `3, 4, 5`.
+ * <!--- KNIT example-delay-01.kt -->
```

Comment:
It would be nice if these tests could be numbered automatically without interfering with the source.
In general, I do no mind having Knit-specific directives in our markdown, but this documentation will likely be read from the IDEA rather than from a webpage with processed markdown, so reducing the number of MD tags would be nice

Reply:
They _are_ automatically numbered. When writing those directives you can use any number or even just a placeholder like `<!--- KNIT example-delay-num.kt -->` and when you run `gradlew knit` it will renumber them sequentially and will update the corresponding source files.